src/test/java/org/fedoraproject/javapackages/validator/MainTmtTest.java

```diff
@@ -150,7 +149,7 @@ void testCustomConfig() throws Exception {
         assertTrue(readResult("results.yaml").contains("result: pass"), "result is pass");
 
         assertEquals(2, theArgs.size());
-        assertEquals("foo", theArgs.get(0));
+        assertEquals("foo", theArgs.getFirst());
```

Comment:
In this single case I would question the consistency.

Reply:
ACK. Fixed.